### PR TITLE
Adding a DB powered changelog

### PR DESCRIPTION
**Pull request by kent**
This PR introduces a changelog for all tenants.

* **DB**: Updated Prisma to include a new changelog table
* **Cached Fetching:** Fetched and cached the queries based on prev pagination code
* **User Interface:** Added components for front-end
* **Multi-Tenant Support:** Scoped calls for multi-tenants
* **Removal of Legacy CSS**: Removed old styles
* **Sensible Defaults**: Added defaults

### CODE REVIEW

File: src/components/Changelog/ChangelogList.tsx [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+  return (
+    <div>
+      <InfiniteScroll
+        hasMore={meta.has_next}
+        pageStart={0}
+        loadMore={loadMore}
+        loader={
+          <div key={0}>
+            <HStack
```
Comment (andreitr) [inline src/components/Changelog/ChangelogList.tsx:76]: In the future, let's use raw tailwind styles instead of HStack VStack components. FYI

**Review by andreitr — APPROVED** [review]
LGTM

File: src/components/Changelog/ChangelogList.tsx (hunk 1 above)
Comment (kent) [reply src/components/Changelog/ChangelogList.tsx:76]: Oh nice, old habits die hard. Sounds good.

File: src/components/Changelog/ChangelogList.tsx [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+        <h2 className="mt-4 text-3xl font-bold text-gray-900">
+          Times are not a changin&apos;
+        </h2>
+        <p className="mt-2 text-lg text-gray-600">
+          It seems nothing has happened on the change log yet.
+        </p>
+        <div className="mt-8">
+          <Image
+            alt="Vacationing changelog"
+            src="https://i.pinimg.com/originals/97/52/18/97521834afab9745868c144cfe189b69.gif"
```
[the remark this answers is not in the document]
Comment (kent) [reply src/components/Changelog/ChangelogList.tsx:59]: Do the kids even know who Bob Dylan is?

**Pull request merged**